README.md

```diff
@@ -112,41 +122,58 @@ the page.
 ### Constructor
 
 ```ts
+
 new KoaWebMonetization(opts: Object | void): KoaWebMonetization
 ```
 
 Create a new `KoaWebMonetization` instance.
 
 - `opts.plugin` - Supply an ILP plugin. Defaults to using Moneyd.
 - `opts.maxBalance` - The maximum balance that can be associated with any user. Defaults to `Infinity`.
-
+- `opts.receiveEndpointUrl` - The endpoint in your Koa route configuration that specifies where a user pays streams PSK packets to your site. Defaults to `/__monetizer/{id}` where `{id}` is the server generated ID (stored in the browser as a cookie).
+- `opts.cookieName` - The cookie key name for your server generated payer ID. Defaults to `__monetizer`.
+- `opts.cookieOptions` - Cookie configurations for Koa. See [Koa ctx setting cookies options](http://koajs.com/) for more details! Only defaults are `httpOnly: false`
 ### Receiver
 
 ```ts
 instance.receiver(): Function
 ```
 
-Returns a koa middleware for setting up Interledger payments with
+Returns a Koa middleware for setting up Interledger payments with
 [SPSP](https://github.com/sharafian/ilp-protocol-spsp) (used in Web
 Monetization).
 
-The endpoint on which this is attached must contain `:id` in the path.
+### Client constructor options
```

Comment:
It might be helpful to document how best to import the client library into the page

Reply:
I was thinking that perhaps we should just have a client library by itself but its configuration has to be the same as its server so I've made it as clear as possible in the example.